Automattic/wp-calypso
# Site Transfers: Add/site transfer UI for pending domain transfer

Related to https://github.com/Automattic/dotcom-forge/issues/2553

## Proposed Changes

This pr adds support for the use case where the user has a pending domain transfer, as mentioned in the above issue

## Testing Instructions

1. Create a simple site
2. Navigate to `Settings > General > Transfer your site`
3. Negate this [this line](https://github.com/Automattic/wp-calypso/pull/77505/files#diff-fb48d30f501ae2da3a2bbd4f2b12ce7c1780f8c0cbb5394be797fba892775771R26) 
4. Ensure that you see a component like the below:

![image](https://github.com/Automattic/wp-calypso/assets/497103/61627f41-dfbf-43a4-8172-e66b39f3b2f7)

NOTE:
The 4th step is not the best way to test this, however, after discussions with @katinthehatsite, I would need to have a custom domain for at least 60 days before it can be transferred away, which I don't.
In case someone has a custom domain, please try to transfer it and test that this component is shown when the site transfer is in process.


## Pre-merge Checklist
